test/plugins/cordova.test.js

```diff
@@ -144,7 +144,7 @@ describe('auth0.plugins.cordova', function() {
           },
           appStatus: null,
           refreshToken: 'kajshdgfkasdjhgfas',
-          state: 'theState',
+          state: null,
```

Comment:
why did you remove the state check on this tests?

Reply:
because all the tests assume a certain behavior from transactionManager, which is a hell to test. So I changed them to use the simplest flow possible and  wrote specific tests for the transaction manager itself.